lib/frame.js

```diff
@@ -0,0 +1,66 @@
+module.exports = class Frame {
+  constructor (maxFramePins = 10, maxFrameRolls = 2, bonusRollsOnStrike = 2, bonusRollsOnSpare = 1) {
+    this._rolls = []
+    this._bonusRolls = []
+    this._maxFramePins = maxFramePins
```

Comment:
Detail - but would the number of frames be better as constants since they don't change?